identity-service/src/tracer.js

```diff
@@ -0,0 +1,53 @@
+const {
+  trace,
+  diag,
+  DiagConsoleLogger,
+  DiagLogLevel
+} = require('@opentelemetry/api')
+
+const { registerInstrumentations } = require('@opentelemetry/instrumentation')
+const { NodeTracerProvider } = require('@opentelemetry/sdk-trace-node')
+const { Resource } = require('@opentelemetry/resources')
+const {
+  SemanticResourceAttributes: ResourceAttributesSC
+} = require('@opentelemetry/semantic-conventions')
+const { HttpInstrumentation } = require('@opentelemetry/instrumentation-http')
+const { ExpressInstrumentation } = require('@opentelemetry/instrumentation-express')
+const { BunyanInstrumentation } = require('@opentelemetry/instrumentation-bunyan')
+
+// Not functionally required but gives some insight what happens behind the scenes
+diag.setLogger(new DiagConsoleLogger(), DiagLogLevel.INFO)
```

Comment:
unclear what this does still - pls document

Reply:
removing